src/Model/Model_System_Report.php

```diff
@@ -98,25 +99,25 @@ public function get_report_structure(): array {
 				'tables'       => [
 					[
 						'title'        => $language['table1'],
-						'title_export' => $language['table1'],
+						'title_export' => $title_export_prefix . $language['table1'],
```

Comment:
The problem here is `$language['table1']` is still displaying translated text via `esc_html__( 'PHP', 'gravity-forms-pdf-extended' )`. You'll have to hardcode `PHP` instead. The same goes for the other `title_exports`.